### ISSUE TITLE
hello, JWT encode error in windows

### ISSUE THREAD

**Issue by ltfafei** [hahwul/jwt-hack#28]
JSON Unmarshal Error

![image](https://user-images.githubusercontent.com/43526141/147476213-bca4b8aa-fcb6-4c66-963f-17fe6a59df79.png)

**Comment by hahwul** [comment]
Hi @ltfafei 
It seems that the quote character is a value other than `"`(%22). 
Is it the same too if you copy and use the contents below?

```
jwt-hack encode '{"a":"z"}'
```

**Comment by ltfafei** [comment]
> Hi @ltfafei It seems that the quote character is a value other than `"`(%22). Is it the same too if you copy and use the contents below?
> 
> ```
> jwt-hack encode '{"a":"z"}'
> ```

Yes, i get the same error, this problem in Windows, Linux not have this problem.

![image](https://user-images.githubusercontent.com/43526141/147620588-09475262-fa1e-4cdb-8ef7-6af845f5ecd7.png)
